.travis.yml

```diff
@@ -3,8 +3,12 @@ language: python
 python:
     - '3.6'
 
+before_install:
+    - pip install git-lint pylint pycodestyle yamllint docutils html-linter
+    - git reset --soft ${TRAVIS_COMMIT_RANGE%...*} && git lint
+
 install:
-    - pip install -q -r requirements-dev.txt .
+    - pip install -r requirements.txt -r requirements-dev.txt -e .
```

Comment:
good catch here! Travis should be testing the prod version, not only the dev version

Reply:
I'm thinking that maybe the requirements.txt is implicated by the `coverage run --source "$(basename "$PWD")" setup.py test`?